### DISCUSSION TITLE
Bind on elytra

### DISCUSSION THREAD

**Discussion by Entity2702** [Geolykt/EnchantmentsPlus#102] [Q&A]
Can i get bind on book to put it on elytra?

**Comment by Geolykt** [comment]
In theory yes, see https://github.com/Geolykt/EnchantmentsPlus/blob/4xx/src/main/java/de/geolykt/enchantments_plus/enchantments/Bind.java#L33 and https://github.com/Geolykt/EnchantmentsPlus/blob/4xx/resources/magicCompat.yml#L329. Should you believe that it is not possible, then feel free to open up an issue.

**Reply by Entity2702** [reply]
Maybe it is possible, but i've enchanted nearly 4 stack of books, maybe more, and i couldn't get this enchantment. Chances of getting are very small or it's impossiple to get. But I will be still trying

**Reply by SonicCraft** [reply]
try changing the rarity?

**Reply by Entity2702** [reply]
> try changing the rarity?

You mean enchantment level? I will try it.

**Reply by Entity2702** [reply]
There is also bug that I can't add for example book with prot. 4 and decapitation 4 to my sword. I thank that gane thinks it's book with only prot. ant as you know you can not add prot. to sword.

**Reply by Entity2702** [reply]
> try changing the rarity?

Thanks a lot. It worked.

**Reply by Geolykt** [reply]
> There is also bug that I can't add for example book with prot. 4 and decapitation 4 to my sword. I thank that gane thinks it's book with only prot. ant as you know you can not add prot. to sword.

Yeah, the anvil merging code is not really working on some systems. I have a half-done third refractor in the works, but it will take a while till that one reaches production
